src/Auth0.php

```diff
@@ -291,6 +294,37 @@ public function renew(
         }
     }
 
+    /**
+     * Return an object representing the current session state (including id token, access token, access token expiration, refresh token and user data) without triggering an authorization flow. Returns null when session data is not available.
+     */
+    public function getState(): ?object
+    {
+        $user = $this->state->getUser();
+
+        if (! $user) {
+            return null;
+        }
+
+        $token = $this->state->getIdToken();
+        $accessToken = $this->state->getAccessToken();
+        $accessTokenExpiration = (int) $this->state->getAccessTokenExpiration();
+        $accessTokenExpired = false;
+        $refreshToken = $this->state->getRefreshToken();
+
+        if (time() >= $accessTokenExpiration) {
```

Comment:
would it make sense to replace

`$accessTokenExpired = false;` with `$accessTokenExpired = time() >= $accessTokenExpiration;` ?

Reply:
Yeah, I like that, I'll update